### DISCUSSION TITLE
Windows Hello authentication after closing the program

### DISCUSSION THREAD

**Discussion by Liub0myr** [keepassxreboot/keepassxc#8399] [Ideas]
Is it possible to make Windows Hello authentication available **after closing the program/restarting the PC**?
Entering a long password every time for the first unlock takes a lot of time. Another thing is a short PIN or biometrics.
Windows Hello stores data in the TPM (if it is available), so it will be impossible to remove the SSD and hack the database through PIN brute force.

**Comment by droidmonkey** [comment]
Yes we are investigating the security and usability , this is tracked in another issue
